Fix issue "You must supply a layout_width attribute" in fragment_user_recoverable_auth_exception.xml

```
[SUMMARY]
android.view.InflateException: Binary XML file line #92 in com.flowcrypt.email:layout/fragment_user_recoverable_auth_exception: Binary XML file line #92: You must supply a layout_width attribute.
java.lang.UnsupportedOperationException: Binary XML file line #92: You must supply a layout_width attribute.


[STACK_TRACE]
android.view.InflateException: Binary XML file line #92 in com.flowcrypt.email:layout/fragment_user_recoverable_auth_exception: Binary XML file line #92: You must supply a layout_width attribute.
Caused by: java.lang.UnsupportedOperationException: Binary XML file line #92: You must supply a layout_width attribute.
        at android.content.res.TypedArray.getLayoutDimension(TypedArray.java:830)
        at android.view.ViewGroup$LayoutParams.setBaseAttributes(ViewGroup.java:8644)
        at android.view.ViewGroup$MarginLayoutParams.<init>(ViewGroup.java:8842)
        at androidx.constraintlayout.widget.ConstraintLayout$LayoutParams.<init>(ConstraintLayout.java:2910)
        at androidx.constraintlayout.widget.ConstraintLayout.generateLayoutParams(ConstraintLayout.java:1934)
        at androidx.constraintlayout.widget.ConstraintLayout.generateLayoutParams(ConstraintLayout.java:486)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:1123)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1082)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:1124)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1082)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:680)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:532)
        at com.flowcrypt.email.databinding.FragmentUserRecoverableAuthExceptionBinding.inflate(FragmentUserRecoverableAuthExceptionBinding.java:112)
        at com.flowcrypt.email.ui.activity.fragment.UserRecoverableAuthExceptionFragment.inflateBinding(UserRecoverableAuthExceptionFragment.kt:56)
        at com.flowcrypt.email.ui.activity.fragment.UserRecoverableAuthExceptionFragment.inflateBinding(UserRecoverableAuthExceptionFragment.kt:53)
        at com.flowcrypt.email.ui.activity.fragment.base.BaseFragment.onCreateView(BaseFragment.kt:69)
        at [androidx.fragment.app](http://androidx.fragment.app/).Fragment.performCreateView(Fragment.java:3104)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentStateManager.createView(FragmentStateManager.java:524)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentStateManager.moveToExpectedState(FragmentStateManager.java:261)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentStore.moveToExpectedState(FragmentStore.java:113)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentManager.moveToState(FragmentManager.java:1424)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentManager.dispatchStateChange(FragmentManager.java:2968)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentManager.dispatchViewCreated(FragmentManager.java:2879)
        at [androidx.fragment.app](http://androidx.fragment.app/).Fragment.performViewCreated(Fragment.java:3129)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentStateManager.createView(FragmentStateManager.java:552)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentStateManager.moveToExpectedState(FragmentStateManager.java:261)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentStore.moveToExpectedState(FragmentStore.java:113)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentManager.moveToState(FragmentManager.java:1424)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentManager.dispatchStateChange(FragmentManager.java:2968)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentManager.dispatchActivityCreated(FragmentManager.java:2886)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentController.dispatchActivityCreated(FragmentController.java:263)
        at [androidx.fragment.app](http://androidx.fragment.app/).FragmentActivity.onStart(FragmentActivity.java:351)
        at [androidx.appcompat.app](http://androidx.appcompat.app/).AppCompatActivity.onStart(AppCompatActivity.java:246)
        at com.flowcrypt.email.ui.activity.BaseActivity.onStart(BaseActivity.kt:94)
        at com.flowcrypt.email.ui.activity.MainActivity.onStart(MainActivity.kt:140)
        at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1435)
        at android.app.Activity.performStart(Activity.java:8195)
        at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3807)
        at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
        at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:173)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2296)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:246)
        at android.app.ActivityThread.main(ActivityThread.java:8447)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:596)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
```

--- a/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/fragment/isolation/incontainer/UserRecoverableAuthExceptionFragmentInIsolationTest.kt
+++ b/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/fragment/isolation/incontainer/UserRecoverableAuthExceptionFragmentInIsolationTest.kt
@@ -1,0 +1,64 @@
+/*
+ * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+ * Contributors: DenBond7
+ */
+
+package com.flowcrypt.email.ui.fragment.isolation.incontainer
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.MediumTest
+import com.flowcrypt.email.R
+import com.flowcrypt.email.base.BaseTest
+import com.flowcrypt.email.rules.AddAccountToDatabaseRule
+import com.flowcrypt.email.rules.AddPrivateKeyToDatabaseRule
+import com.flowcrypt.email.rules.ClearAppSettingsRule
+import com.flowcrypt.email.rules.RetryRule
+import com.flowcrypt.email.rules.ScreenshotTestRule
+import com.flowcrypt.email.ui.activity.fragment.UserRecoverableAuthExceptionFragment
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.rules.TestRule
+import org.junit.runner.RunWith
+
+/**
+ * @author Denis Bondarenko
+ *         Date: 8/3/22
+ *         Time: 6:20 PM
+ *         E-mail: DenBond7@gmail.com
+ */
+@MediumTest
+@RunWith(AndroidJUnit4::class)
+class UserRecoverableAuthExceptionFragmentInIsolationTest : BaseTest() {
+  val addAccountToDatabaseRule = AddAccountToDatabaseRule()
+
+  @get:Rule
+  var ruleChain: TestRule = RuleChain
+    .outerRule(RetryRule.DEFAULT)
+    .around(ClearAppSettingsRule())
+    .around(addAccountToDatabaseRule)
+    .around(AddPrivateKeyToDatabaseRule())
+    .around(ScreenshotTestRule())
+
+  @Before
+  fun launchFragmentInContainerWithPredefinedArgs() {
+    launchFragmentInContainer<UserRecoverableAuthExceptionFragment>()
+  }
+
+  @Test
+  fun testBaseInfo() {
+    onView(
+      withText(
+        getResString(
+          R.string.reconnect_your_account,
+          getResString(R.string.app_name), addAccountToDatabaseRule.account.email
+        )
+      )
+    ).check(matches(isDisplayed()))
+  }
+}

--- a/FlowCrypt/src/main/res/layout/fragment_user_recoverable_auth_exception.xml
+++ b/FlowCrypt/src/main/res/layout/fragment_user_recoverable_auth_exception.xml
@@ -92,6 +92,7 @@
         <Button
             android:id="@+id/buttonReconnect"
             style="@style/AppWidget.Button.OutlinedButton.Big"
+            android:layout_width="0dp"
             android:layout_marginBottom="@dimen/default_margin_content"
             android:paddingLeft="0dp"
             android:paddingRight="0dp"
@@ -102,6 +103,7 @@
 
         <Button
             android:id="@+id/buttonLogout"
+            android:layout_width="0dp"
             style="@style/AppWidget.Button.OutlinedButton.Big"
             android:layout_marginBottom="@dimen/default_margin_content_big"
             android:paddingLeft="0dp"

